src/main/resources/plugin.yml

```diff
@@ -4,7 +4,6 @@ author: Cyprias
 authors: [Jikoo]
 description: Limit entities in a chunk.
 version: ${project.version}
-api-version: 1.13
```

Comment:
Any reason this was removed? I don't see a reason to enable the compatibility layer if it's not needed.

Reply:
Nope. Chuck er back in. My bad. :)